pythonforandroid/toolchain.py

```diff
@@ -179,6 +180,7 @@ def build_dist_from_args(ctx, dist, args):
             ctx, dist.recipes, bs,
             blacklist=blacklist
         ))
+    assert(len(set(build_order).intersection(set(python_modules))) == 0)
```

Comment:
- ahah  you're still using parenthesis on `assert`
- also I would rather skip the `len` check and just do `== set()`

Reply:
lol! done